src/CONST.ts

```diff
@@ -1720,6 +1721,7 @@ const CONST = {
     MAX_64BIT_LEFT_PART: 92233,
     MAX_64BIT_MIDDLE_PART: 7203685,
     MAX_64BIT_RIGHT_PART: 4775807,
+    INVALID_CATEGORY_NAME: '###',
```

Comment:
Can you help me for my knowledge we don't allow `###` (3 #) but allow the other counts?

Reply:
I'm not too sure haha This is a 9 year old check we have in the API 🤷

Reply:
Okay, let's leave it then ;)